src/main/java/com/alkemy/ong/service/impl/CommentServiceImpl.java

```diff
@@ -30,4 +32,14 @@ public Long saveComment(CommentDto dto) {
 			repository.save(comment);
 			return comment.getId();
 	}
+
+	public List<CommentDto> getAllCommentsByPost(Long id){
+		return newsRepository.findById(id).map((news)->{
+			List<Comment>commentList = repository.findByNewsId(id);
+			return mapper.toListCommentDto(commentList);
+		}).orElseThrow(()->{
+			throw new NewNotFoundException();
```

Comment:
We do not return NOT_FOUND when a list of resources is needed, just return an empty array and a status code 200.